src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const DAYS_PRICE = 40;
+  const SHORT_DISCOUNT = 20;
+  const LONG_DISCOUNT = 50;
```

Comment:
It is better to keep static constants outside the function definition so that they are not created anew each time the function is called.